.buildkite/install-lmcache.sh

```diff
@@ -16,6 +16,6 @@ set -xe
 pip install -e third_party/torchac_cuda
 pip install -e .
 
-git clone https://github.com/LMCache/lmcache-server
+git clone --branch test https://github.com/LMCache/lmcache-server
```

Comment:
We might still need to use `dev` or `main` branch in the future.
Let's change it back after lmcache server's PR is merged. Opened a new issue #59 for this